test/suggestion-builder.js

```diff
@@ -0,0 +1,63 @@
+import test from 'tape'
+import suggestionBuilder from '../src/suggestion-builder'
+
+
+test('suggestion Builder | term suggest', (t) => {
+    t.plan(1)
+
+    const result = suggestionBuilder().suggest('term', 'products', { text: 'testing' })
+
+    t.deepEqual(result.getSuggestions(), {
+        'suggest_term_products': {
```

Comment:
As mentioned above, the test cases here should be expecting:
```
        suggest: {
            'suggest_term_products': {
                text: 'testing',
                term: {
                    field: 'products',
                }
            }
        }
```

Reply:
This just builds the inner suggestions, when we actually call `bb.suggest()` it will add it inside a `suggest` object on the BB top level